src/index.ts

```diff
@@ -26,10 +26,10 @@ function updateCanvasSize() {
 window.addEventListener('resize', function () { updateCanvasSize(); redraw(); });
 updateCanvasSize();
 
-canvas.onmousedown = function (e) {
+canvas.onpointerdown = function (e) {
     if (e.button === 0) {
         hasCapture = true;
-        //canvas.setPointerCapture((e as PointerEvent).pointerId);
+        canvas.setPointerCapture((e as PointerEvent).pointerId);
```

Comment:
I'm guessing the cast is no longer needed?

Reply:
Ah, you're right. I'll remove the cast.

Reply:
Cast removed in c23386f.